dataplane/src/packet_processor/ingress.rs

```diff
@@ -69,30 +69,39 @@ impl Ingress {
         }
     }
 
+    #[tracing::instrument(level = "trace")]
     fn interface_ingress_eth_non_local<Buf: PacketBufferMut>(
         &self,
-        _interface: &Interface,
+        interface: &Interface,
         dst_mac: Mac,
         packet: &mut Packet<Buf>,
     ) {
-        let nfi = self.name();
         /* Here we would check if the interface is part of some
         bridge domain. But we don't support bridging yet. */
-        trace!("{nfi}: Recvd frame for mac {dst_mac} (not for us)");
+        trace!(
+            "{nfi}: Recvd frame for mac {dst_mac} (not for us) (interface {ifname})",
+            nfi = self.name(),
+            ifname = interface.name
+        );
         packet.done(DoneReason::MacNotForUs);
     }
 
+    #[tracing::instrument(level = "trace")]
     fn interface_ingress_eth_bcast<Buf: PacketBufferMut>(
         &self,
-        _interface: &Interface,
+        interface: &Interface,
         packet: &mut Packet<Buf>,
     ) {
         let nfi = self.name();
```

Comment:
If the purpose of this change is not taking references that may not be used if the tracing is disabled, I believe you missed this one?